aodndata/templates/soop_xbt_nrt_bufr_nc_template.json

```diff
@@ -0,0 +1,117 @@
+    "LONGITUDE": 1
+  },
+  "_variables": {
+    "TIME": {
+      "_datatype": "float64",
+      "units": "days since 1950-01-01 00:00:00 UTC",
+      "calendar": "gregorian",
+      "standard_name": "time",
+      "long_name": "time",
+      "axis": "T",
+      "valid_min": 0.0,
+      "valid_max": 90000.0
+    },
+    "LATITUDE": {
+      "_FillValue": 999999.0,
+      "_datatype": "float64",
+      "standard_name": "latitude",
+      "long_name": "latitude",
+      "units": "degrees_north",
+      "axis": "Y",
+      "valid_min": -90.0,
+      "valid_max": 90.0,
+      "reference_datum": "WGS84 coordinate reference system"
+    },
+    "LONGITUDE": {
+      "_FillValue": 999999.0,
+      "_datatype": "float64",
+      "standard_name": "longitude",
+      "long_name": "longitude",
+      "units": "degrees_east",
+      "axis": "X",
+      "valid_min": -180.0,
+      "valid_max": 180.0,
+      "reference_datum": "WGS84 coordinate reference system"
+    },
+    "DEPTH":{
+      "_datatype": "float64",
+      "_dimensions": ["DEPTH"],
+      "axis": "Z",
+      "long_name": "depth",
+      "positive": "down",
+      "reference_datum": "sea surface",
+      "standard_name": "depth",
+      "units": "m",
+      "ancillary_variables": "DEPTH_quality_control",
+      "valid_max": 12000.0,
+      "valid_min": -5.0,
+      "_data": 0.6
+    },
+    "DEPTH_quality_control": {
+      "_dimensions": ["DEPTH"],
+      "_datatype": "b",
+      "standard_name": "depth status_flag",
+      "long_name": "quality flag for sea_water_temperature",
+      "quality_control_conventions": "IMOS standard flags",
+      "_FillValue": 63,
+      "valid_min": 0,
+      "valid_max": 9,
+      "flag_values": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9],
+      "flag_meanings": "No_QC_performed Good_data Probably_good_data Bad_data_that_are_potentially_correctable Bad_data Value_changed Not_used Not_used Not_used Missing_value"
+    },
+    "TEMP": {
+      "_dimensions": ["DEPTH"],
+      "_datatype": "float64",
+      "standard_name": "sea_water_temperature",
+      "long_name": "sea_water_temperature",
+      "units": "Celsius",
+      "ancillary_variables": "TEMP_quality_control",
+      "_FillValue": 999999.0,
+      "valid_min": 5.0,
+      "valid_max": 30.0,
+      "coordinates": "TIME LATITUDE LONGITUDE DEPTH"
+    },
+    "TEMP_quality_control": {
+      "_dimensions": ["DEPTH"],
+      "_datatype": "b",
+      "standard_name": "sea_water_temperature status_flag",
+      "long_name": "quality flag for depth",
+      "quality_control_conventions": "IMOS standard flags",
+      "_FillValue": 63,
+      "valid_min": 0,
+      "valid_max": 9,
+      "flag_values": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9],
+      "flag_meanings": "No_QC_performed Good_data Probably_good_data Bad_data_that_are_potentially_correctable Bad_data Value_changed Not_used Not_used Not_used Missing_value"
+    }
+  },
+  "project": "Integrated Marine Observing System (IMOS)",
+  "Conventions": "CF-1.6,IMOS-1.4",
```

Comment:
Now that you're using `ImosTemplate`, you no longer need to have these fixed, standard global attributes here (Conventions, project, acknowledgement, etc...) They're already defined by default ([here](https://github.com/aodn/python-aodntools/blob/master/aodntools/ncwriter/imos_global.json)).

If you do specify some of these here, your values will override the default ones.

Reply:
I'll just leave the Conventions one in the template as I feel like if we decide to update the ```ImosTemplate``` value to a new version, this could break this pipeline for example

Reply:
Good point.
I guess this one should be called `ImosTemplate1_4`, and then if we have a newer version based on updated conventions, it would then be e.g. `ImosTemplate1_5` (though I don't see that happening any time soon...)